packages/spacecat-shared-rum-api-client/src/utils.js

```diff
@@ -11,6 +11,26 @@
  */
 
 import { context as h2, h1 } from '@adobe/fetch';
+import { utils } from '@adobe/rum-distiller';
+import { classifyTrafficSource } from './common/traffic.js';
+
+export const DELIMITER = '≡';
```

Comment:
can't we put these stuff (DELIMITER, generateKey, and the loadBundles)  into the original rum-distiller lib?